python_src/src/lamp_py/performance_manager/l1_rt_trips.py

```diff
@@ -373,7 +389,6 @@ def backup_rt_static_trip_match(
     # static trip are needed.
     first_stop_static_sub = (
         sa.select(
-            static_trips_sub.c.static_version_key,
```

Comment:
guessing this is redundant since all these subqueries already have the static version key baked in?

Reply:
Yup